dotnet-authserver/src/TeacherIdentity.AuthServer/Services/UserSearch/NamesSynonymService.cs

```diff
@@ -0,0 +1,55 @@
+namespace TeacherIdentity.AuthServer.Services.UserSearch;
+
+public class NamesSynonymService : INamesSynonymService
+{
+    private readonly Dictionary<string, HashSet<string>> namesLookup = new Dictionary<string, HashSet<string>>();
+
+    public NamesSynonymService()
+    {
+        Initialise();
```

Comment:
We typically use US English
```suggestion
        Initialize();
```

Reply:
Done